notebooks/15_polygon_fill.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #| exporti
</pre><p>@joshuacortez suggest changing the nbdev directive <code>exporti</code> to <code>export</code> so that <code> scanline_fill</code> and <code>interpolate_x</code>  are    added to <code>__all__</code> default module exports as well as showing up as documentation.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/geowrangler/pull/245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
thanks butch i addressed in the latest commit